conans/cli/api/conan_api.py

```diff
@@ -32,18 +33,14 @@ def __init__(self, cache_folder=None):
 
         # Remotes management
         self.remotes = RemotesAPI(self)
-
         # Search recipes by wildcard and packages filtering by configuracion
         self.search = SearchAPI(self)
-
         # Get latest refs and list refs of recipes and packages
         self.list = ListAPI(self)
-
         self.profiles = ProfilesAPI(self)
-
         self.install = InstallAPI(self)
-
         self.graph = GraphAPI(self)
+        self.export = ExportAPI(self)
```

Comment:
I don't like the semantics of `api.export.export()`. Probably the same for the install. I don't know if we might move them all to a `command` subapi.

Reply:
Agree. There is still a global discussion about the API and the CLI for different commands. Here I am trying just to "modernize" a bit the code, moving it to another subapi would be straightforward.